<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>981</width>
      <height>641</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>forth_page_1.webp</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>811</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: rgba(255, 255, 255, 0.9); /* White background with 80% opacity */
    border-radius: 50px; /* Rounded corners */
    padding: 20px; /* Optional: Add some padding inside the box */
    margin: 20px; /* Optional: Add margin around the box */
    color: #000000; /* Text color inside the box */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">FONT-SIZE:30PX;</string>
    </property>
    <property name="text">
     <string>EIGEN VALUE AND VECTOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #50a9b8, stop:1 #62ba74); /* Blue to black as main theme */
    border-radius: 25px; /* Optional: for rounded corners */
    border: none; /* Optional: remove border */
    color: white; /* Text color */
    padding: 10px 20px; /* Optional: adjust padding */
    font-size: 16px; /* Optional: adjust font size */
    transition: background 10s; /* Super slow transition */
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #62ba74, stop:1 #50a9b8); /* Black to blue on hover */
}
</string>
    </property>
    <property name="text">
     <string>SOLVE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>820</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background: transparent; /* Makes the background transparent */
    font-size: 36px; /* Adjusts the font size */

    color: black; /* Sets the text color */
}
</string>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                background-color: transparent;
                border: none; /* Remove default border */
                padding: 0;
                icon-size: 100px; /* Adjust icon size if needed */
            }
            QPushButton:hover {
                background-color: rgba(0, 0, 0, 0.1); /* Slightly darkened background */
                border: none;
            }
            QPushButton::icon {
                padding: 0;
                border-image: url(path_to_your_image.png); /* Set image */
            }
            QPushButton:hover::icon {
                border-image: url(path_to_your_hover_image.png); /* Blurred or darkened image on hover */
            }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/DALL_E_2024-09-04_17.45.07_-_Create_several_spherical_buttons_with_a_back_arrow_symbol_in_a_style_that_matches_the_colorful_and_creative_physics-themed_image_filled_with_equations__1_-modified-removeb.png</normaloff>../../Downloads/DALL_E_2024-09-04_17.45.07_-_Create_several_spherical_buttons_with_a_back_arrow_symbol_in_a_style_that_matches_the_colorful_and_creative_physics-themed_image_filled_with_equations__1_-modified-removeb.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Instructions</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
font:bold 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(234, 234, 234,0.7);

border: 2px solid rgba(0,0,0); 
border-radius:25px;
padding-top:10px;
padding-left:7px;
padding-bottom:1px;</string>
    </property>
    <property name="text">
     <string>• Use space to separate each element.
• Press the &quot;+&quot; button to add a new row.
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0.5);
border-bottom: 3px solid rgb(106, 0, 106);
font: 11pt &quot;MS Shell Dlg 2&quot;;
border-radius: 25px;
padding-left:10px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>enter elements like:-    5 6 7 8 9 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(204, 204, 204, 0.4);
border-radius:20px;
font: 18pt &quot;MS Shell Dlg 2&quot;;
border: 1px dashed rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="shortcut">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
